src/CommonITILObject.php

```diff
@@ -2365,7 +2365,7 @@ public function pre_updateInDB()
                 $this->updates[]                 = "time_to_resolve";
                 $this->fields['time_to_resolve'] = $sla->computeDate(
                     $this->fields['date'],
-                    $this->fields["sla_waiting_duration"]
+                    (int) $this->fields["sla_waiting_duration"]
```

Comment:
Since this is done in `computeDate`, this is no longer required here (also for others)